### PR TITLE
Additional: added password strength method.

### PR DESCRIPTION
**Pull request by NiranjanaK**
Hi @jzaefferer 
I implemented the password strength method, Please take a look. 

It's regarding the issue https://github.com/jzaefferer/jquery-validation/issues/1683

### CODE REVIEW

File: src/additional/passwordstrength.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+$.validator.addMethod("pwcheck", function(value) {
```
Comment (staabm) [inline src/additional/passwordstrength.js]: should also be named `password-strength`

**Comment by staabm** [comment]
please add a unit test

**Comment by staabm** [comment]
on a second though: I guess nearly every application/corporation has different requirements/rules on password security.

I dont think this should be part of the library.

**Comment by staabm** [comment]
@NiranjanaK please discuss the above mentioned issues at first. the PR will not be merged until we reached a consent.

**Comment by NiranjanaK** [comment]
@staabm Thanku for your reviews.

**Comment by staabm** [comment]
As mentioned earlier, I dont think this makes sense to be bundled with the lib. Password strenght rules are very different, depending on the problem domain.

**Pull request closed without merging**